[DOCS] Reformat compound word token filters (#49006)
elastic/elasticsearch: docs/reference/analysis/tokenfilters/dictionary-decompounder-tokenfilter.asciidoc

--------------------------------------------------

/////////////////////
[source,console-result]
--------------------------------------------------
{
  "tokens" : [
    {
      "token" : "Donaudampfschiff",
      "start_offset" : 0,
      "end_offset" : 16,
      "type" : "<ALPHANUM>",
      "position" : 0
    },
    {
      "token" : "Donau",
      "start_offset" : 0,
      "end_offset" : 16,
      "type" : "<ALPHANUM>",
      "position" : 0
    },
    {
      "token" : "dampf",
      "start_offset" : 0,
      "end_offset" : 16,
      "type" : "<ALPHANUM>",
      "position" : 0
    },
    {
      "token" : "schiff",
      "start_offset" : 0,
      "end_offset" : 16,
      "type" : "<ALPHANUM>",
      "position" : 0
    }
  ]
}
--------------------------------------------------
/////////////////////

[[analysis-dict-decomp-tokenfilter-configure-parms]]
==== Configurable parameters

`word_list`::
+
--
(Required+++*+++, array of strings)
A list of subwords to find in the token stream. If found, the subword is
included in the token output.

Either this or the `word_list_path` parameter is required.
--

`word_list_path`::
+
--
(Required+++*+++, string)

> Path to a file containing a list of subwords to find in the token stream. If

Reviewer: ```suggestion
Path to a file that contains a list of subwords to find in the token stream. If
```
Getting into picky grammar-tics that are more prescriptive than truly necessary.

found, the subword is included in the token output.

This path must be absolute or relative to the `config` location. The file must

> be UTF-8 encoded. Each token in the file must be separated by a line break.

Reviewer: I'd be inclined to combine the first two sentences:

>This path must be absolute or relative to the `config` location, and the file must be UTF-8 encoded.

> Either this or the `word_list` parameter is required.

Reviewer: ```suggestion
Either this parameter or `word_list` must be specified.
```
